onnxruntime/test/shared_lib/test_nontensor_types.cc

```diff
@@ -28,120 +28,66 @@ struct RelAllocations {
 
 TEST_F(CApiTest, CreateGetVectorOfMapsInt64Float) {  // support zipmap output type seq(map(int64, float))
   // Creation
-  std::unique_ptr<MockedOrtAllocator> default_allocator(std::make_unique<MockedOrtAllocator>());
```

Comment:
Nice that so much of boiler plate can be eliminated. Thanks!

Reply:
The only ugly part was the line that was meant to generate an error. Want me to put a try/catch around it?

Reply:
Try/catch done!